components/logins/src/encryption.rs

```diff
@@ -26,24 +26,140 @@
 // multiple layers, from the app saying "sync now" all the way down to the
 // low level sync code.
 // To make life a little easier, we do that via a struct.
+//
+// Consumers of the Login component have 3 options for setting up encryption:
```

Comment:
It might make sense to offer some guidance about which of the 3 options consumers might choose, even if it requires speculation about the future (eg, you might mention how you picture desktop using it even though that doesn't exist etc)